web/src/composables/useLogs.ts

```
@@ -4129,6 +4161,177 @@
     }
   };
 
+  const setSelectedStreams = (value: string) => {
+    try {
+      const parsedSQL = fnParsedSQL();
+
+      if (!parsedSQL) {
+        console.error("Failed to parse SQL query:", value);
+        throw new Error("Invalid SQL syntax");
+        return;
+      }
```

Biome

[error] This code is unreachable



(lint/correctness/noUnreachable)